admin-dev/themes/new-theme/js/pages/category/index.ts

```diff
@@ -95,11 +95,11 @@ $(() => {
   new Serp(
     {
       container: '#serp-app',
-      defaultTitle: 'input[name^="category[name]"]',
-      watchedTitle: 'input[name^="category[meta_title]"]',
-      defaultDescription: 'textarea[name^="category[description]"]',
-      watchedDescription: 'textarea[name^="category[meta_description]"]',
-      watchedMetaUrl: 'input[name^="category[link_rewrite]"]',
+      defaultTitle: 'input[name*="category[name]"]',
+      watchedTitle: 'input[name*="category[meta_title]"]',
+      defaultDescription: 'textarea[name*="category[description]"]',
+      watchedDescription: 'textarea[name*="category[meta_description]"]',
+      watchedMetaUrl: 'input[name*="category[link_rewrite]"]',
       multiLanguageInput: `${translatorInput.localeInputSelector}:not(.d-none)`,
```

Comment:
Before SERP didn't work properly in root category since all the names are like `root_category[link_rewrite][1]` and `name^` finds only inputs which start with `category[link_rewrite]`. `name*` finds any inputs with `category[link_rewrite]` which includes `root_category[link_rewrite]`